ground/src/main/res/drawable/cluster_marker.xml

```diff
@@ -15,14 +15,23 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item>
+    <shape android:shape="oval">
+      <solid android:color="#4B6DDD81" />
```

Comment:
Should we add a comment indicating this is ~29% opacity clusterColor?

Reply:
Sure. I'll do you one better; since the color of the cluster balloons doesn't change, I now moved the color codes to `colors.xml` and make them static so that we don't need to tint them at runtime.